### ISSUE TITLE
 On resizing the page to 200%, content of  "Identity Provider" get distorted.

### ISSUE THREAD

**Issue by utkarshUU1** [microsoft/fluentui#15033]
Package version(s): 7.24.0
OS: Version 2004(OS Build 19041.388)
Browser: Edge(Chrome)Version 86.0.594.1 (Official build) dev (64-bit)

User Experience:
On resizing the page to 200%, if content of the page get distorted then visually impaired user get effected and might not be able to understand the content to interact with it.

Please provide a reproduction of the issue in a codepen:

https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist

https://codepen.io/utkars_12334ss/pen/oNxyKQY?editable=true%3Dhttps%3A%2F%2Fdeveloper.microsoft.com%2Fen-us%2Ffluentui

![image](https://user-images.githubusercontent.com/71307703/93173302-f4cd7400-f749-11ea-9b3c-a337178ae005.png)

Using DetailsList Example in codepen, We can see some of the columns are not visible and we cannot see them

![image](https://user-images.githubusercontent.com/71307703/93173045-999b8180-f749-11ea-8c92-1de2dca0d883.png)

Actual behavior:
On resizing the page to 200%, content of  "DetailsList " get distorted, only 2 colums are visible

Expected behavior:
On resizing the page to 200%, content of  "DetailsList " should be properly visible , all the columns should be visible

**Comment by paulgildea** [comment]
@utkarshUU1 Thanks for the feedback.

@smhigley - Any ideas on what the right behavior should be here?

**Comment by smhigley** [comment]
Hm, interesting. I think the problem here is "proper behavior" is highly contextual. For example, it would be completely fine to hide columns on smaller screen sizes, so long as there were still a mechanism to see all the data in context (e.g. another page/view with the full table).

As long as developers can opt out and decide to show all columns all the time, it doesnt seem like a bad thing to provide the ability to collapse columns for smaller viewports. Maybe a docs change addressing `isCollapsible` and reflow/zoom access?
